activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   `travel/travel_to` travel time helpers, now raise on nested calls, 
+     as this can lead to confusing time stubbing.
+       
+     Instead of:
+     
+         travel_to 2.days.from_now do
+           # 2 days from today
+           travel_to 3.days.from_now do
+             # 5 days from today
```

Comment:
Hehe, we discussed and I showed some examples from our production Application actually that does this. It is a Task Management Application, and the test verifies notifications. This is in case of nested cases, where you would like to simulate nested way to achieve cases, and do calls to contexts from already executing operation.